fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/temp/PlannerExpression.java

```diff
@@ -107,5 +112,39 @@
         }
         return null;
     }
+
+    default PlannerGraphBuilder<InternalPlannerGraphProperty.Node, InternalPlannerGraphProperty.Edge> showYourself() {
+        final InternalPlannerGraphProperty.Node root = new InternalPlannerGraphProperty.Node(getClass().getSimpleName());
+        return PlannerGraph.builder(root);
+    }
+
+    default String show() {
+        try {
+            final PlannerGraph<InternalPlannerGraphProperty.Node, InternalPlannerGraphProperty.Edge> plannerGraph =
+                    Objects.requireNonNull(acceptPropertyVisitor(new InternalPlannerGraphProperty()));
+            final URI uri = InternalPlannerGraphProperty.createHtmlLauncher(Objects.requireNonNull(plannerGraph));
+            Desktop.getDesktop().browse(uri);
```

Comment:
It would be really nice if we didn't need to "pollute" a data structure interface like `PlannerExpression` with this `URI` and `Desktop` stuff. Is it possible to keep the nice IntelliJ integration while moving this logic into some kind of "visualizer" or other auxiliary class?

At a bare minimum, I think we could probably have this call a `VisualizerRegistry` or something like that.